docs/release/old-release-removal.md

```diff
@@ -0,0 +1,90 @@
+Old Release Series Removal Plan
+===============================
+
+In order to reduce clutter and disk usage on our repositories and build system,
+we should remove older OSG Software release series.  This will result in
+packages from those series becoming unavailable both to users, and to developers.
+
+
+Policy
+------
+
+We should delete a release series only when needing to download a package from
+that series is improbable.  I propose the following rule:
```

Comment:
I think you have buy-in for this policy so you can remove this intro paragraph

Reply:
I disagree; sure, we know we have buy-in now, but if we're keeping this doc around for years, then we'll need a justification we can look back at.

Reply:
Ok, then combine the previous two sentences (the users and developers bit is unnecessary, anyway):

> This will result in packages from those series becoming unavailable so we should only remove a release series when those packages are no longer needed.